File: ads3/poo/pratica_aulas/Semana1/aula04/Program.java
import java.util.Scanner;
public class Program{
	public static void main(String[] args){
		Scanner ler = new Scanner(System.in);
		String nome;
		double valorHora;
		int horasTrabalhadas;
		double descontos;

		System.out.print("Informe o nome: ");
		nome = ler.nextLine();

		System.out.print("Informe o valor da hora: ");
		valorHora = ler.nextDouble();

		System.out.print("Informe a quantidade de horas trabalhadas: ");
		horasTrabalhadas = ler.nextInt();

		System.out.print("Informe o valor descontado: ");
		descontos = ler.nextDouble();

		System.out.printf("\n\nFolha de pagamento: %s\n",nome);
		System.out.printf("%d x R$%.2f = R$%.2f\n",horasTrabalhadas,valorHora,valorHora*horasTrabalhadas);
                System.out.printf("Descontos: R$%.2f\n",descontos);
                System.out.printf("Total devido: %.2f\n",(valorHora*horasTrabalhadas)-descontos);

		ler.close();

	}
}

File: ads3/poo/pratica_aulas/Semana1/aula04/Program.cpp
#include <iostream>

int main(int argc, char *argv[]){
	using namespace std;

	double valorHora;
	int horasTrabalhadas;
	double descontos;
	string nome;

	cout << "Informe o nome: ";
	cin >> nome;
	setbuf(stdin,NULL);
	cout << "Informe o valor da hora: ";
	cin >> valorHora;

	cout << "Informe a quantidade de horas trabalhadas: ";
	cin >> horasTrabalhadas;

	cout << "Informe o valor descontado: ";
	cin >> descontos;

	cout << "\n\nFolha de pagamento: " << nome << endl;
	cout << horasTrabalhadas << " x R$" << valorHora << "= R$" << (valorHora*horasTrabalhadas) << endl;
        cout << "Descontos: R$" << descontos << endl;
        cout << "Total devido: R$" << ((valorHora*horasTrabalhadas)-descontos) << endl;

	return 0;
}
